### ISSUE TITLE
Vector sink has unusable marker lines

### ISSUE THREAD

**Issue by tpiepho-synapse** [gnuradio/gnuradio#3110]
The `VectorDisplayPlot` has two marker lines, named `LowerIntensityLevel` and `UpperIntensityLevel`.    They default to visible, yet these is no way to make any use of them, as neither the `vector_sink` interface nor the `vector_sink_f_impl` class provide any interface to them.

Seems like they should default to off, or just be removed, if there is no way to ever give them a value other than zero.

**Comment by marcusmueller** [comment]
That's right, still no way to manipulate these things from `gr::qtgui::vector_sink`

Should be removed.
